README.md

```diff
@@ -120,6 +120,69 @@ import utc from '@momentjs/moment/timezones/utc/'; // Note trailing slash
 // Error: folders cannot be imported (there is no index.* magic)
 ```
 
+### 2. Imports Field
+
+> **To avoid conflict with `node_modules` packages, the current proposal prefixes all imports with `#name`, so that the fact that an alias is being imported is clear. Whether this restriction is maintained in the final proposal, or what symbol is used is still TBD.**
```

Comment:
we don’t have to bike she’d here, but i prefer `~`

Reply:
Definitely open to that.